spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -40,29 +41,34 @@ Spell Actions:
 * [ ] Rates match
   * [ ] Compare against [IPFS](https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6)
   * [ ] Check manually via `make rates pct=<pct>` (e.g. pct=0.75, for 0.75%)
-  * [ ] Variable visibility declared as internal
+  * [ ] Variable name conforms to `X_PT_Y_Z_PCT_RATE` (e.g. `ZERO_PT_SEVEN_FIVE_PCT_RATE` for 0.75%)
+  * [ ] Variable visibility declared as `internal`
+  * [ ] State mutability declared as `constant`
 * [ ] Math matches
   * [ ] Internal Precision
-    * [ ] `WAD = 10**18`
-    * [ ] `RAY = 10**27`
-    * [ ] `RAD = 10**45`
+    * [ ] `WAD = 10 ** 18`
+    * [ ] `RAY = 10 ** 27`
+    * [ ] `RAD = 10 ** 45`
     * [ ] Ensure they match with [ds-math](https://github.com/dapphub/ds-math/blob/master/src/math.sol) and the [Numerical Ranges](https://github.com/makerdao/dss/wiki/Numerical-Ranges#notation)
     * [ ] Variable visibility declared as internal
+    * [ ] State mutability declared as `constant`
   * [ ] Units
-    * [ ] `HUNDRED  = 10**2`
-    * [ ] `THOUSAND = 10**3`
-    * [ ] `MILLION  = 10**6`
-    * [ ] `BILLION  = 10**9`
-    * [ ] Ensure they match with [config](https://github.com/makerdao/spells-mainnet/blob/master/src/test/config.sol)
+    * [ ] `HUNDRED  = 10 ** 2`
+    * [ ] `THOUSAND = 10 ** 3`
+    * [ ] `MILLION  = 10 ** 6`
+    * [ ] `BILLION  = 10 ** 9`
     * [ ] Variable visibility declared as internal
+    * [ ] State mutability declared as `constant`
+    * [ ] Ensure they match with [config](https://github.com/makerdao/spells-mainnet/blob/master/src/test/config.sol)
+    
 * [ ] Deployed Contracts
   * [ ] Verified on etherscan
   * [ ] Optimizations match Repo
   * [ ] `GNU AGPLv3` license
   * [ ] Constructor args ok (e.g. `vat`, `dai`, `dog`, ...)
     * [ ] Match [ChainLog](https://chainlog.makerdao.com/)
   * [ ] Wards ok (pause proxy relied, deployer denied)
-    * [ ] Check whether the contract requires to rely the ESM in the spell (in order to allow de-authing the pause proxy during Emergency Shutdown, via `denyProxy`).
+    * [ ] `MCD_ESM` is relied (as approved by GovAlpha) in order to allow de-authing the pause proxy during Emergency Shutdown, via `denyProxy`.
```

Comment:
I would suggest that those 6 occurrences of "`MCD_ESM` is relied" have to be publicly approved by the govalpha in this repo, not just casually [on discord](https://discord.com/channels/893112320329396265/897483518316265553/1105146557453574286) (which is semi-private)

Reply:
FYI: this was already approved by govalpha here https://github.com/makerdao/pe-checklists/pull/12#issuecomment-1540017564